argon2.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "Louis Roché <louis@louisroche.net>"
+authors: "Louis Roché <louis@louisroche.net>"
+homepage: "https://github.com/Khady/ocaml-argon2"
+dev-repo: "git+https://github.com/Khady/ocaml-argon2.git"
+bug-reports: "https://github.com/Khady/ocaml-argon2/issues"
+license: "MIT"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune"
+  "ctypes" {>= "0.4.1"}
+  "ctypes-foreign"
+  "result"
+]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs "@install" "@doc" {with-doc}]
```

Comment:
is that equivalent to?
```
["dune" "build" "-p" name "-j" jobs]
["dune" "build" "-p" name "-j" jobs "@doc"] {with-doc}
```

Reply:
The end-result is equivalent, but there's only ever one invocation of dune, so it's faster (academically faster for a small library, of course). If a predicate isn't satisfied for a parameter, then it's omitted.

Reply:
However, you remind me that I've made a mistake - `"odoc" {with-doc}` should be in the `depends` now